test/src/org/apache/jmeter/protocol/http/util/TestHTTPArgument.java

```diff
@@ -81,7 +81,7 @@ public void testEncoding() throws Exception {
             assertEquals("value_ here", arg.getValue());
             assertEquals("name.%3F", arg.getEncodedName());
             assertEquals("value_+here", arg.getEncodedValue());
-            arg.setAlwaysEncoded(false); // by default, name/value are encoded on fetch
+            arg.setAlwaysEncoded(false); // by default, name/value are encoded on fetch 
```

Comment:
Is this change intentional?

Reply:
This change was done in order to not include any unnecessary changes [trailing whitespace removal]. If you want I can make a different commit and remove this trailing whitespace. Please suggest.

Reply:
Created a new commit to remove this trailing whitespace.